pkg/tests/common/appstudio.go

```diff
@@ -19,26 +23,35 @@ var (
 var _ = framework.CommonSuiteDescribe("Red Hat App Studio common E2E", func() {
 	defer g.GinkgoRecover()
 	commonController, err := commonCtrl.NewSuiteController()
-	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	Expect(err).NotTo(HaveOccurred())
 
 	g.Context("Argo CD", func() {
 		for _, component := range AppStudioComponents {
 			g.It(component+" status", func() {
 				componentStatus, err := commonController.GetAppStudioComponentStatus(component, AppStudioComponentsNamespace)
-				gomega.Expect(err).NotTo(gomega.HaveOccurred())
-				gomega.Expect(componentStatus.Health.Status).To(gomega.Equal(health.HealthStatusHealthy))
+				Expect(err).NotTo(HaveOccurred())
+				Expect(componentStatus.Health.Status).To(Equal(health.HealthStatusHealthy))
 
 			})
 		}
 	})
+	defer g.GinkgoRecover()
```

Comment:
This is already called at lone 24 - why call it here again?
TBH, I haven't used Ginkgo nor Gomega for years, but I pretty like the functionality provided by `testing.Cleanup` method https://pkg.go.dev/testing#B.Cleanup - it can be called outside of the actual test for example by some helper function.

Reply:
My mistake. I did some tests in my local machine. I will remove this `defer g.GinkgoRecover()`